app/assets/stylesheets/widgets/home_page/hero_panel.scss

```diff
@@ -130,6 +151,24 @@
       height: auto;
     }
   }
+  .tagline {
+    font-family: $futura-regular;
```

Comment:
Is there a reason this is `$futura-regular` instead of `$font-stack-heading`?

Reply:
It looks to be just a mistake. I will update.